deps.edn

```diff
@@ -205,33 +206,36 @@
   ;;    (dev/start!)
   :dev
   {:extra-deps
-   {clj-http-fake/clj-http-fake  {:mvn/version "1.0.4"
-                                  :exclusions  [slingshot/slingshot]}
-    clj-kondo/clj-kondo          {:mvn/version "2023.09.07"}                ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
-    cloverage/cloverage          {:mvn/version "1.2.4"}
-    com.gfredericks/test.chuck   {:mvn/version "0.2.14"}                    ; generating strings from regexes (useful with malli)
-    djblue/portal                {:mvn/version "0.46.0"}                    ; ui for inspecting values
-    io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    io.github.metabase/hawk      {:sha "9c97bcb6d4de116325e651b40973fd0a75b7ae21"}
-    jonase/eastwood              {:mvn/version "1.4.0"                      ; inspects namespaces and reports possible problems using tools.analyzer
-                                  :exclusions
-                                  [org.ow2.asm/asm-all]}
-    lambdaisland/deep-diff2      {:mvn/version "2.10.211"}                  ; way better diffs
-    methodical/methodical        {:mvn/version "0.15.1"}                    ; drop-in replacements for Clojure multimethods and adds several advanced features
-    org.clojure/algo.generic     {:mvn/version "0.1.3"}
-    pjstadig/humane-test-output  {:mvn/version "0.11.0"}
-    reifyhealth/specmonstah      {:mvn/version "2.1.0"
-                                  :exclusions  [org.clojure/clojure
-                                                org.clojure/clojurescript]} ; lets you write test fixtures that are clear, concise, and easy to maintain (clojure.spec)
-    ring/ring-mock               {:mvn/version "0.4.0"}                     ; creating Ring request maps for testing purposes
-    talltale/talltale            {:mvn/version "0.5.13"}}                   ; generates fake data, useful for prototyping or load testing
+   {com.clojure-goes-fast/clj-async-profiler {:mvn/version "1.1.1"}
```

Comment:
this diff is too large to see what is going on. Ideally we don't reformat everything

Reply:
If we want to align the opening `{:mvn/version`, we need to indent the rest of the form since `com.clojure-goes-fast/clj-async-profiler` is long.

Reply:
This was the only change in this block `com.clojure-goes-fast/clj-async-profiler {:mvn/version "1.1.1"}` and the rest was manually reformatted to align the values.

Reply:
We have a couple other deps which have the same issue, and the solution was to put the version map on the next line to keep it aligned. See `org.apache.logging.log4j/log4j-layout-template-json` for example.

Reply:
I tend to add `?w=1` to an URL when there are a lot of whitespace changes.